docs/user-guide/01-blueprint-reference.md

```diff
@@ -64,7 +64,7 @@ The `packages` and `modules` lists contain objects with a `name` and optional `v
 
 *Currently there are no differences between packages and modules in `osbuild-composer`. Both are treated like an rpm package dependency.*
 
-> When using virtual provides as the package name the version glob should be `*`. And be aware that you will be unable to `freeze` the blueprint. This is because the provide will expand into multiple packages with their own names and versions.
+> When using virtual providers as the package name the version glob should be `*`. And be aware that you will be unable to `freeze` the blueprint. This is because the provider will expand into multiple packages with their own names and versions.
```

Comment:
I think this should actually be "provides". In virtual packages, this keyword is used to list all the packages provided by a virtual package.
Maybe we could make it clearer by using markdown for monospace formatting `provides` to make it visually stand out as a technical term?

Reply:
Thank you for clarifying this. I will fix it and add the correct markup.